programs/locked-voter/src/instructions/move_escrow.rs

```diff
@@ -0,0 +1,110 @@
+use crate::*;
+use anchor_spl::token;
+/// Accounts for [voter::move_escrow].
+#[derive(Accounts)]
+pub struct MoveEscrow<'info> {
```

Comment:
I think this flow would create many security issues:
1. smart_wallet can't move tokens to an arbitrary escrow, that must be defined by old_escrow owner
2. move tokens to the new escrow create different behaviours for new escrow: some escrows are max_locked, some escrows are not, if we enforce all new escrows must be max_locked then it may affect on current unlocking process of new escrow 

My suggestion: if smart_wallet resolves, we just set a cooldown duration in old escrow, after the cooldown, owner of old escrow can withdraw tokens back to the token_account that they defined in previous step.